ledger-service/http-json/src/main/resources/logback.xml

```diff
@@ -1,10 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <configuration>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <!-- encoders are assigned the type
-             ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
-        </encoder>
+        <if condition='isDefined("LOG_FORMAT_JSON")'>
+            <then>
+                <encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
+            </then>
+            <else>
+                <!-- encoders are assigned the type
+                    ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+                <encoder>
+                    <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg %replace(, context: %marker){', context: $', ''} %n</pattern>
```

Comment:
This solution is fine but please note that you need to apply it to all `logback.xml` files that are used by default in all the components that use the contextualized logger, otherwise the logging context will be lost there.

Reply:
Great, will apply it to the other files then :)

Reply:
OK did that, hope I caught all projects which are affected.